docs/glossary.md

```diff
@@ -30,4 +30,4 @@
 + **Productions server**: A server on AppSpot. Can be the live server or a staging server.
 + **Spec**: System Specification.
 + **Staging green**: All test are passing against you own deployment on the GAE.
-+ **Staging server**: The server instance you created on AppSpot.
\ No newline at end of file
++ **Staging server**: The server instance you created on AppSpot.
```

Comment:
@rishikksh20 "Look same" is quite a dangerous word to use: things like different encodings (e.g for line endings, for whitespace) or trailing whitespaces are considered by Git as differences. I suspect that this one is caused by different line endings (although I'm not sure why), can you try and revert it?